core/kernel/otp_stubs.c

```diff
@@ -17,11 +17,13 @@
  * The default implementation just sets it to a constant.
  */
 
+#ifdef CFG_INSECURE
```

Comment:
Add an explicit statement like `This default implementation cannot be used when CFG_INSECURE is disabled.` in the inline comment above?
Ditto for `plat_rng_init()`.